--- kopeninghours-23.08.5/PyKOpeningHours/CMakeLists.txt.orig	2024-02-09 22:59:08.000000000 +0100
+++ kopeninghours-23.08.5/PyKOpeningHours/CMakeLists.txt	2025-05-26 05:13:08.626663629 +0200
@@ -22,7 +22,7 @@
 
 # Set up the libraries and header search paths for this target
 target_link_libraries(PyKOpeningHours PUBLIC ${Boost_LIBRARIES} ${PYTHON_LIBRARIES} KOpeningHours)
-target_include_directories(PyKOpeningHours PRIVATE ${Boost_INCLUDE_DIR} ${PYTHON_INCLUDE_DIRS})
+target_include_directories(PyKOpeningHours PRIVATE ${BOOST_INCLUDE_DIR} ${PYTHON_INCLUDE_DIRS})
 
 set(_install_dir lib/python${PYTHONLIBS_VERSION_MAJOR}.${PYTHONLIBS_VERSION_MINOR}/site-packages)
 
